fastavro/errors.py

```diff
@@ -0,0 +1,24 @@
+from .util import path_string
+
+
+class UnknownType(Exception):
+    def __init__(self, name):
+        super(UnknownType, self).__init__(name)
+        self.name = name
+
+
+class AvroValueError(ValueError):
+    pass
+
+    @classmethod
```

Comment:
Why is this a `classmethod` and not a regular `__init__`?
Also class method which are alternate initializers are usually named `from...`